dietza/whats-cookin
# Feature/scripts development

# Pull Request Template:

#### What's this PR do?
Finalizes scripts functionality.
Adds functionality to user sidebar.
Removes vestigial code.

#### Where should the reviewer start?
Open index.html and unit test the functionality.

#### How should this be manually tested?
(see above)
Run index.html in browser and check the console to ensure no errors are presented.

#### Any background context you want to provide?
n/a

#### What are the relevant tickets?
n/a

#### Screenshots (if appropriate):
n/a

#### Questions
n/a
